data/assets/scene/solarsystem/planets/saturn/hyperion/hyperion.asset

```diff
@@ -26,18 +26,14 @@ local Hyperion = {
     SegmentsPerPatch = 64,
     Layers = {},
     Labels = {
-      Enable = false,
+      -- @TODO (2021-05-21, emmbr26) These labels do not seem to be visible
+      Enabled = false,
       FileName = labelsPath .. "/hyperion.labels",
       LabelAlignmentOption = "Horizontally", -- or Circularly
```

Comment:
Since the label variables are in the `Labels` table, I think we would drop the "Label" prefix from them?
`LabelAlignmentOption` -> `AlignmentOption`
`LabelsFontSize` -> `FontSize`

etc

Reply:
I agree

Reply:
Done, for `RenderableLabels` and `GlobeLabelsComponent`